client/src/components/SystemStatus/SystemStatusQueueTable/SystemStatusQueueTable.module.scss

```
@@ -0,0 +1,36 @@
+.root {
+  --table-border: 1px solid black;
+
+  margin: 0;
+  width: 100%;
+
+  /* system */
+  tr > *:nth-child(1) {
+    width: 25%;
+  }
+  /* status */
+  tr > *:nth-child(2) {
+    width: 30%;
+
+    span {
+      width: 25%;
+    }
+  }
+  /* free */
+  tr > *:nth-child(3) {
+    width: 15%;
+  }
+  /* running */
+  tr > *:nth-child(4) {
+    width: 15%;
+  }
+  /* waiting */
+  tr > *:nth-child(5) {
+    width: 15%;
+  }
+}
```

To let core styles identify width automatically while still making the pill not no take up the entire cell. Having some issues with the table on smaller screens with these settings
```suggestion
.root {
  --table-border: 1px solid black;

  margin: 0;
  width: 100%;
  /* status */
  tr > *:nth-child(2) {
    width: 20%;

    span {
      width: 50%;
    }
  }
}
```

---

Thanks for this suggestion. I'll test it out!